apps/fuzzyw/fuzzyw.app.js

```diff
@@ -31,113 +31,129 @@
     /*LANG*/"ten to *$2",
     /*LANG*/"five to *$2"
   ]
-};
+  };
 
-let text_scale = 4;
-let timeout = 2.5*60;
-let drawTimeout;
-let animInterval;
-let time_string = "";
-let time_string_old = "";
-let time_string_old_wrapped = "";
+  let text_scale = 4;
+  let timeout = 2.5*60;
+  let drawTimeout;
+  let animInterval;
+  let time_string = "";
+  let time_string_old = "";
+  let time_string_old_wrapped = "";
+  let settings = {};
 
-let loadSettings = function() {
-  settings = require("Storage").readJSON(SETTINGS_FILE,1)|| {'showWidgets': false};
-};
+  let loadSettings = function() {
+    settings = require("Storage").readJSON(SETTINGS_FILE,1)|| {'showWidgets': false, 'animate': true};
+  };
 
-let queueDraw = function(seconds) {
-  let millisecs = seconds * 1000;
-  if (drawTimeout) clearTimeout(drawTimeout);
-  drawTimeout = setTimeout(function() {
-    drawTimeout = undefined;
-    draw();
-  }, millisecs - (Date.now() % millisecs));
-};
+  let queueDraw = function(seconds) {
+    let millisecs = seconds * 1000;
+    if (drawTimeout) clearTimeout(drawTimeout);
+    drawTimeout = setTimeout(function() {
+      drawTimeout = undefined;
+      draw();
+    }, millisecs - (Date.now() % millisecs));
+  };
 
-let getTimeString = function(date) {
-  let segment = Math.round((date.getMinutes()*60 + date.getSeconds() + 1)/300);
-  let hour = date.getHours() + Math.floor(segment/12);
-  f_string = fuzzy_string.minutes[segment % 12];
-  if (f_string.includes('$1')) {
-    f_string = f_string.replace('$1', fuzzy_string.hours[(hour) % 12]);
-  } else {
-    f_string = f_string.replace('$2', fuzzy_string.hours[(hour + 1) % 12]);
-  }
+  let getTimeString = function(date) {
+    let segment = Math.round((date.getMinutes()*60 + date.getSeconds() + 1)/300);
+    let hour = date.getHours() + Math.floor(segment/12);
+    // add "" to load into RAM due to 2v21 firmware .replace on flashstring issue
+    f_string = ""+fuzzy_string.minutes[segment % 12]; 
```

Comment:
```suggestion
    let f_string = ""+fuzzy_string.minutes[segment % 12]; 
```

to avoid the lint warning that came with the latest commit.